modules/audio_splitter.py

```diff
@@ -0,0 +1,44 @@
+from pydub import AudioSegment
+from pydub.silence import split_on_silence
+import os
+import time
+import logging
```

Comment:
separe os imports padrões do python dos imports de terceiros. Algo desse tipo:
```suggestion
import os
import time
import logging

from pydub import AudioSegment
from pydub.silence import split_on_silence
```
Isso vem do padrão PEP8: https://www.python.org/dev/peps/pep-0008/#imports